File: Medium/Java/1828. Queries on Number of Points Inside a Circle.java
class Solution {
    public int[] countPoints(int[][] points, int[][] queries) {
        int[] nums = new int[queries.length];
        int i = 0;

        for(int[] q : queries){
            int x0 = q[0];
            int y0 = q[1];
            int r = q[2];
            for(int[] p : points){
                int x = p[0];
                int y = p[1];
                int dx = x - x0;
                int dy = y - y0;
                if((dx * dx) + (dy * dy) <= (r * r))
                    nums[i]++;
            }
            i++;
        }

        return nums;
    }
}

File: Medium/Python/1828. Queries on Number of Points Inside a Circle.py
class Solution(object):
    def countPoints(self, points, queries):
        nums = [0] * len(queries)
        i = 0

        for q in queries:
            x0 = q[0]
            y0 = q[1]
            r = q[2]
            for p in points:
                x = p[0]
                y = p[1]
                dx = x - x0
                dy = y - y0
                if (dx * dx) + (dy * dy) <= (r * r):
                    nums[i] += 1
            i += 1
        
        return nums
